CSOC_Documentation.md

```diff
@@ -2,54 +2,54 @@
 
 ![CSOC_networkdiag](CSOC_networkdiag_V1.png)
 
-CDIS uses a Commons Services Operations Center (CSOC) AWS account for accessing the AWS accounts associated with all of the Gen3 commons they operate. Operators of other Gen3 data commons may choose to have a separate CSOC account or operate their Gen3 data commons within the same AWS account. There are two major functionality associated with the CSOC account :
- 
+CDIS uses a Commons Services Operations Center (CSOC) AWS account for accessing the AWS accounts associated with all of the Gen3 commons they operate. Operators of other Gen3 data commons may choose to have a separate CSOC account or operate their Gen3 data commons within the same AWS account. There are two major functionality associated with the CSOC account:
+
 ## A. CSOC Logging
 Centralize logging for all commons in other AWS accounts.
 
-https://github.com/uc-cdis/cloud-automation/blob/master/tf_files/modules/cdis-aws-common-logging/README.md
+https://github.com/uc-cdis/cloud-automation/blob/master/tf_files/aws/modules/common-logging/README.md
 
 ## B. CSOC Centralized Management VPC
-CDIS launched  `csoc_main_vpc [CIDR - 10.128.0.0/20]`, using a vpc wizard with two subnets - `csoc_main_vpc_pub1 [ public subnet - 10.128.1.0/24]` and `csoc_main_vpc_priv1 [private subnet - 10.128.2.0/24]` in the AWS CSOC account. 
+CDIS launched  `csoc_main_vpc [CIDR - 10.128.0.0/20]`, using a vpc wizard with two subnets - `csoc_main_vpc_pub1 [ public subnet - 10.128.1.0/24]` and `csoc_main_vpc_priv1 [private subnet - 10.128.2.0/24]` in the AWS CSOC account.
 
 ## Components of CSOC VPC:
 
-	1.	OpenVPN server VM 
-	2.	Admin VM for the CSOC account 
+	1.	OpenVPN server VM
+	2.	Admin VM for the CSOC account
 	3.	Admin VM for data commons account
 	4.	Squid Proxy VM
 	5.	VPC Peering
 
 
 
-## 1. OpenVPN server VM 
+## 1. OpenVPN server VM
 
-An openvpn server has been manually configured on an instance in the public subnet of the CSOC VPC. It is a one time spun-up and rather a permanent resource. One needs to connect to the  CSOC VPN server to access the admin VMs in the CSOC. The PlanX admin can request for  an openVPN credentials. Upon request an email will be sent with a link to the QR code, openvpn files and instructions on how to set-up the VPN client. 
+An openvpn server has been manually configured on an instance in the public subnet of the CSOC VPC. It is a one time spun-up and rather a permanent resource. One needs to connect to the  CSOC VPN server to access the admin VMs in the CSOC. The PlanX admin can request for  an openVPN credentials. Upon request an email will be sent with a link to the QR code, openvpn files and instructions on how to set-up the VPN client.
```

Comment:
This is slightly out of date - I'm not sure to what extent it's important to document here, but FYI - we actually run 2 VPN servers in our CSOC:
* dev vpn - gives access to the admin vm for our dev environments
* prod vpn - gives access to admin vm's for prod environments

To access controls for a particular environment a user must be able to authenticate to the VPN, and also have ssh access to the admin vm.